xla/service/gpu/matmul_utils.h

```diff
@@ -70,6 +70,10 @@ absl::StatusOr<Shape> GetBatchRowColumnShape(
 absl::StatusOr<bool> CanFoldTransposeOperandIntoDot(const HloInstruction& dot,
                                                     int64_t operand_idx);
 
+// Returns true if the matrices are too small to benefit from cublas/triton.
```

Comment:
This docstring is misleading; this will return true if the size of each (unbatched) matmul is below the threshold, independently of whether cuBLAS and Triton would be good codegen solutions.

Maybe "Returns true if the combined size of the (unbatched) operands is smaller than the given threshold."?